webpack.config.js

```diff
@@ -0,0 +1,92 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+const WatchMissingNodeModulesPlugin = require('react-dev-utils/WatchMissingNodeModulesPlugin');
+const DashboardPlugin = require('webpack-dashboard/plugin');
+
+const PATHS = {
+    charts: path.join(__dirname, 'src/charts'),
+    build: path.join(__dirname, 'build'),
+};
+
+const commonConfig = {
+    entry: {
+        stackedArea: PATHS.charts + '/stackedArea/stackedAreaComponent.js',
```

Comment:
we'll replace this with PATHS.charts eventually right?

Reply:
Yeahh, I guess we will need an object, as we do currently in Britecharts